test/docker/compose.go

```diff
@@ -318,7 +318,7 @@ func (d *Compose) WithGenerativeFriendliAI(apiKey string) *Compose {
 }
 
 func (d *Compose) WithGenerativePaLM(apiKey string) *Compose {
-	d.withPaLMApiKey = apiKey
+	d.withGoogleApiKey = apiKey
```

Comment:
just wondering if method name should be renamed for consistency

Reply:
I will make a separate PR just for `Generative Google` module and will change that in this PR